src/components/AlertBanner/__tests__/AlertBanner-snapshots.jest.js

```diff
@@ -6,20 +6,6 @@ import AlertBannerButton from "../AlertBannerButton/AlertBannerButton";
 import AlertBannerText from "../AlertBannerText/AlertBannerText";
 import { NOOP } from "../../../utils/function-utils";
 
-jest.mock("../../Button/Button", () => {
```

Comment:
after converting to ts this mock cause errors on running the tests so I thought it can remove in this stage and later we can figure out how to involve mocks in a way that will work with ts

Reply:
Which errors? Why did we need this mock in the first place?

Reply:
the mock return undefined instead of the object which cause exceptions.
We need to solve it but I think it is ok to split it to a separate pr (because this is a bit big already).
I did not implement this tests but in general mock like this in snapshot are used for decrease the dependency of a component in the structure of a component inside it, so snapshot will not break when the button structure is change.
It would be better to have mock here but we have a lot of components which not using mocks for this so I think it is ok to handle this issue separately